pocs/dome/astrohaven.py

```diff
@@ -68,7 +68,11 @@ def is_open(self):
     def open(self):
         self._full_move(Protocol.OPEN_A, Protocol.A_OPEN_LIMIT)
         self._full_move(Protocol.OPEN_B, Protocol.B_OPEN_LIMIT)
-        return self.is_open
+        v = self._read_state_until_stable()
```

Comment:
Not quite sure if the call to open A is blocking or not but since `A_IS_OPEN` is one of the `stable` states it seems like this could open A but not B and still return `True`. Is that intended?

Reply:
Yes, intended in this case: _full_move is "blocking" (i.e. it doesn't return until it accomplishes its goal or times out), so ideally it fully opens A, then fully opens B.

I'm thinking though of changing full_move so that it sends at a faster rate, and only stops after it gets several of the limit responses, as I noticed that the dome this morning was very slightly open (a little light leaking in). I'll do that in a separate PR though.